extensions/azurecore/src/azureResource/utils.ts

```diff
@@ -80,6 +81,10 @@ export class AzureResourceErrorMessageUtil {
 	}
 }
 
+export function getTenantVersion(): vscode.WorkspaceConfiguration {
+	return vscode.workspace.getConfiguration('azure.tenantVersion');
```

Comment:
This is returning a configuration object, which you're then using directly in the strings. I assume you mean to do `vscode.workspace.getConfiguration('azure').get('tenantVersion');`